src/ppx/imgui_impl.cpp

```diff
@@ -187,7 +187,7 @@ Result ImGuiImplDx12::InitApiObjects(ppx::Application* pApp)
     bool result = ImGui_ImplDX12_Init(
         grfx::dx12::ToApi(pApp->GetDevice())->GetDxDevice(),
         static_cast<int>(pApp->GetNumFramesInFlight()),
-        grfx::dx::ToDxgiFormat(pApp->GetSwapchain()->GetColorFormat()),
+        grfx::dx::ToDxgiFormat(pApp->GetUISwapchain()->GetColorFormat()),
```

Comment:
Looks like this doesn't compile, where is GetUISwapchain defined?

Reply:
Sorry about that, didn't realize it wasn't available for non-XR. Fixed.